.circleci/config.yml

```diff
@@ -133,6 +133,43 @@ jobs:
             make migrate
             make check
 
+  # ---- Front-end jobs ----
+  build-front:
+    docker:
+      - image: circleci/node:10
+    working_directory: ~/fun/src/frontend
+    steps:
+      - checkout:
+          path: ~/fun
+      - restore_cache:
+          keys:
+            - v1-front-dependencies-{{ checksum "yarn.lock" }}
+            - v1-front-dependencies-
+      - run:
+          name: Install frontend dependencies
+          command: make install
+      - run:
+          name: Build application styles
+          command: make build-sass
```

Comment:
There is no `Makefile` in `src/frontend`. You should run `yarn sass` or `yarn sass-production` depending what you want.